tests/unit/s2n_cipher_suite_match_test.c

```diff
@@ -781,6 +798,460 @@ int main(int argc, char **argv)
             }
         }
 
+        /* Test ChaCha20 boosting behaviour for sslv2 connection */
```

Comment:
what do you mean with sslv2 connection ?

Reply:
What's really being tested here is calling `s2n_set_cipher_as_sslv2_server` instead of `s2n_set_cipher_as_tls_server`. The `actual_protocol_version` is set to tls 12 to bypass the higher vers match validation; we are still using sslv2 ciphersuites and their formats